### PR TITLE
View shortest paths to gc roots in debugger dialog

### PR DESCRIPTION
**Pull request by nikita-nazarov**

### CODE REVIEW

**Comment by nikita-nazarov** [comment]
@bibaev , please take a look

**Comment by ybedrov** [comment]
Hello,

Have you submitted a contributor license agreement? If not, please follow the steps described at http://www.jetbrains.com/agreements/cla/ to sign it.

File: platform/xdebugger-impl/src/com/intellij/xdebugger/impl/ui/tree/actions/ShowReferringObjectsAction.java [hunk 1]
```diff
@@ -31,9 +32,25 @@ protected void perform(XValueNodeImpl node, @NotNull String nodeName, AnActionEv
                                                    nodeName,
                                                    referrersProvider.getReferringObjectsValue(),
                                                    tree.getValueMarkers(), session, false);
+        XDebuggerTree dialogTree = (XDebuggerTree)dialog.getPreferredFocusedComponent();
+        if (dialogTree != null) {
+          dialogTree.expandNodesOnLoad(treeNode -> isInTopSubTree(treeNode));
```
Comment (bibaev) [inline platform/xdebugger-impl/src/com/intellij/xdebugger/impl/ui/tree/actions/ShowReferringObjectsAction.java:37]: Infinite recursion occurs if an object refers itself

**Comment by nikita-nazarov** [comment]
Merged internally

**Pull request closed without merging**